src/assets/theater-mask.svg

```diff
@@ -0,0 +1,126 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M293.825 208.29C295.088 204.775 299.418 202.819 303.863 203.927C308.212 205.011 311.086 208.633 310.708 212.251C310.779 212.295 310.849 212.34 310.918 212.385C311.384 208.593 308.386 204.824 303.917 203.71C299.351 202.572 294.841 204.601 293.578 208.317C293.66 208.308 293.742 208.299 293.825 208.29ZM293.746 208.523C293.665 208.532 293.584 208.542 293.504 208.551C293.477 208.643 293.451 208.735 293.428 208.829C292.421 212.867 295.515 217.083 300.286 218.273C305.058 219.462 309.769 217.193 310.776 213.154C310.819 212.979 310.855 212.804 310.884 212.629C310.815 212.584 310.747 212.54 310.678 212.496C310.648 212.698 310.609 212.899 310.559 213.1C309.59 216.984 305.03 219.225 300.341 218.056C295.651 216.886 292.677 212.767 293.645 208.883C293.675 208.761 293.709 208.642 293.746 208.523Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M272.732 205.215C277.432 206.451 279.242 206.47 281.154 206.059C283.066 205.648 284.927 205.452 286.502 203.582C287.001 205.312 285.498 206.302 281.21 209.31C276.922 212.318 272.982 211.502 270.903 214.368C270.914 214.024 271.16 211.883 275.288 209.771C279.416 207.658 275.002 208.097 272.732 205.215Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M298.763 210.367C304.031 212.721 306.259 213.809 306.259 213.809C306.259 213.809 293.742 210.368 291.824 214.72C290.941 217.034 294.673 219.267 294.673 219.267C294.673 219.267 298.907 224.377 292.731 223.442C286.554 222.507 285.086 224.354 284.627 224.298C286.609 223.03 287.15 220.93 286.83 218.843C286.51 216.757 285.585 215.877 283.095 214.817C283.193 215.029 287.279 214.361 289.201 212.435C291.371 210.261 294.625 206.079 294.9 206.276C295.133 206.443 293.496 208.012 298.763 210.367Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M286.458 220.575C286.72 216.828 284.004 215.806 283.296 214.922C285.524 215.991 287.587 216.716 288.347 218.698C289.152 220.802 288.996 223.764 284.697 224.217C285.249 223.502 286.301 222.823 286.458 220.575Z" fill="white"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M306.906 198.933C302.717 198.277 300.949 199.887 299.686 201.165C302.817 200.721 303.806 200.555 306.561 201.212C309.316 201.868 310.082 202.815 311.714 204.032C311.825 202.189 311.095 199.59 306.906 198.933Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M286.465 194.208C283.492 192.472 281.423 193.415 279.891 194.201C282.516 194.669 283.356 194.794 285.214 196.135C287.072 197.476 287.259 198.528 288.003 200.057C288.884 198.452 289.437 195.944 286.465 194.208Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M315.464 228.808C312.823 231.209 308.133 234.614 303.928 236.301C307.52 232.147 307.317 231.156 312.709 226.674C318.102 222.192 315.276 217.425 318.454 216.393C320.498 221.466 319.881 224.79 315.464 228.808Z" fill="white"/>
+</g>
+<defs>
+<clipPath id="clip0">
```

Comment:
let's make it more specific like `theater-mask-clip0`. You know why 😄  Other thing is that `clip-path` may cause some conflicts. Might be worth double-checking that.

Also, this illustration is almost the same as `empty-uploads-illustration` (this one doesn't have this circular shadow). Not sure if we should use the same in both places or not

Reply:
removed the clip-path at the end it was not needed and used only theater-mask.svg as they are indeed the same